plugins/Login/lang/en.json

```
@@ -83,6 +86,16 @@
         "CancelPasswordResetRequestRemoved": "The request has been removed",
         "CancelPasswordResetRequestRemovedMessage": "We have recorded that you did not request this password reset. The reset request has been removed, and no changes have been made to your account.",
         "CancelPasswordResetSecurityTip": "For added protection, we recommend enabling two-factor authentication (2FA).",
-        "SecurityTip": "Security Tip"
+        "SecurityTip": "Security Tip",
+        "LoginFromDifferentCountryEmailSubject": "Unusual login activity detected on your Matomo account",
+        "LoginFromDifferentCountryEmailBody01": "We’ve detected a login to your Matomo account from a location that is different from your usual login area.",
+        "LoginFromDifferentCountryEmailBody02": "Login details:",
+        "LoginFromDifferentCountryEmailBody03": "If this was you, no action is needed. If you did not recognise this login attempt, please take the following steps to secure your account:",
+        "LoginFromDifferentCountryEmailBody04": "Reset your password: %1$sreset your password%2$s",
+        "LoginFromDifferentCountryEmailBody05": "Enable Two-Factor Authentication (2FA) for additional security: %1$senable 2FA%2$s",
+        "LoginFromDifferentCountryEmailBody06": "We also recommend reviewing your recent account activity to ensure there are no other unauthorised actions.",
+        "IPAddress": "IP address",
+        "Country": "Country",
+        "DateAndTime": "Date & time"
```

The `&` should be used encoded in English, as weblate will return in encoded for all translations anyway.
```suggestion
        "DateAndTime": "Date &amp; time"
```